javascript/javascript1/week3/smartphone.js

```diff
@@ -0,0 +1,52 @@
+"use strict"
+
+//adding activities
+const activities =[];
+var today  = new Date();
+
+var date=today.toLocaleDateString("en-US"); 
+function addActivity(activity ,duration){
+    activities.push({date,activity ,duration});
+}
+addActivity("youtube",30)
+addActivity("facebook",90)
+addActivity("news",80)
+addActivity("netflix",40)
+console.log(activities);
+
+
+//show my status
+let usageAmount=0;
+let usageLimit=60;
+function showStatus(){
+for(let i=0;i<activities.length;i++){
+    usageAmount=usageAmount+activities[i].duration;
+}
+     if(usageAmount >=usageLimit){
+        console.log("You have reached your limit, no more smartphoning for you!")
+    }
+    else{
+        console.log("You have the limit, you can use smartphone!")
+    }
+
+const totalActivities=activities.length;
+console.log("You have added " +totalActivities+ " activities. They amount to " +usageAmount+ " min. of usage");
+}
+if(activities.length === 0||activities.length===null){
+    console.log("Add some activities before calling showStatus");
+}else{
+    showStatus()
```

Comment:
why do you call  showStatus() inside showStatus() ?

Reply:
i changed this.actually i did a mistake.